lib/gon/spec_helpers.rb

```diff
@@ -5,7 +5,7 @@ module Rails
 
       module ClassMethods
         module GonSession
-          def process(*, **)
+          def process(...)
```

Comment:
This works with Ruby 3.x, but I think it might not work with some versions of Ruby 2.x.
The current master branch has a CI environment that runs tests from Ruby 2.3 to 3.5,
so could you modify it to make sure all of them pass?